pkg/operator/deploy/deploy.go

```diff
@@ -184,18 +184,10 @@ func (o *operator) resources() ([]kruntime.Object, error) {
 		},
 	}
 
-	if o.oc.Properties.NetworkProfile.GatewayPrivateEndpointIP != "" {
-		cluster.Spec.GatewayDomains = append(o.env.GatewayDomains(), o.oc.Properties.ImageRegistryStorageAccountName+".blob."+o.env.Environment().StorageEndpointSuffix)
-	} else {
-		cluster.Spec.InternetChecker = arov1alpha1.InternetCheckerSpec{
-			URLs: []string{
-				fmt.Sprintf("https://%s/", o.env.ACRDomain()),
```

Comment:
From our sync I understood we would still need to populate `spec.InternetChecker`, is that not the case?

Reply:
It's still populated from https://github.com/Azure/ARO-RP/blob/7b70040dd4e23084e4659ec6514ee083e8021842/pkg/operator/deploy/deploy.go#L160-L166

Reply:
This else statement was apparently redundant and not needed

Reply:
thanks for clarifying Ross